server/src/services/users/users.go

```diff
@@ -79,6 +79,9 @@ func (s *UserService) Update(_ context.Context, body dto.UserUpdateRequest) (*dt
 		Avatar: body.Avatar,
 	})
 	s.UpdatedUser(user)
```

Comment:
Maybe we should check it even before we send an update (with`s.UpdatedUser`) to the users.
Additionally, it would be nice to create a log message before returning the error in case we need to debug it in the future.

Reply:
Hard to tell where to log 😅 
We use it differently for different files. But when it is logged in the API layer, it should be sufficient for now :D